calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,147 @@
+---
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI. Because the scanner is not running continuously, it may take some time for results to display in the dashboard, especially if you have many images or large images.
+
+## Before you begin
+
+**Required**
+
+- You must have valid registry credentials
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Google GCR 
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Setting up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a docker image. Simply run `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4` in order to retrieve it.
+
+### Setting up registry scanner
+
+#### Credentials
+
+The registry scanner supports scanning all registries as long as the docker credentials are provided.
+Providing the `DOCKER_USERNAME` and `DOCKER_PASSWORD` lets the registry scanner retrieve images that the credentials have access to. Retrieving images 
+using these credentials will only retrieve images with tags.
+
+If you have a valid /.docker/config.json, you can also mount these while running the registry scanner.
+```
+docker run -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+
+The registry scanner also supports authentication using AZURE and AWS credentials. If you use these credentials, the registry scanner is able to
+retrieve images with no tags associated with them.
```

Comment:
I find this part a bit confusing from a user's perspective. I don't think I would necessarily make the connection that I can provide the registry scanner with either my Docker credentials vs platform specific credentials (ACR or ECR) and get a different set of results. 

It kind of feels a bit wrong to allow that. I realize this isn't the time to open up a can of worms regarding the actual behaviour of the registry scanner, but from a user docs perspective, I think we should try to explain this limitation better. 

E.g. perhaps we could highlight this consideration (of scanning tagged vs untagged images) with a visual call out box. @bmckercher123 I don't know if we have different types of boxes other than the caution type of box. But I think that might be a good way to call this out so the user can better understand the limitation and how to avoid it.

Reply:
Seems like we should call this out in Limitations in Before you begin...

**Limitations**

- The registry scanner is available as an image (not using Tigera operator installation)
- You can scan images from only one of the supported registry platforms/account per instance
- If you use the registry scanner with Docker, only tagged images are scanned. However, if you use the scanner with Amazon ECR or Azure Container Registry, both tagged and untagged images are scanned. 

Then, I still agree we should add a note inline to remind users of this limitation. I would also spell out in the note the implications of canning images without tags. Does it mean a huge number of images will be scanned that the user will have to filter out in the dashboard? We should say something. 

:::note
If you use the registry scanner with AZURE and AWS credentials, the registry scanner will retrieve images with no tags associated with them.
:::